src/workflows-app/SubmissionHistory.test.js

```diff
@@ -0,0 +1,561 @@
+    const mockLeoResponse = jest.fn(() => Promise.resolve(mockAzureApps));
+
+    await Ajax.mockImplementation(() => {
+      return {
+        Cbas: {
+          runSets: {
+            get: getRunSetsMethod,
+          },
+        },
+        Apps: {
+          listAppsV2: mockLeoResponse,
+        },
+      };
+    });
+
+    // Act
+    await act(async () => {
+      render(
+        h(BaseSubmissionHistory, {
+          name: 'test-azure-ws-name',
+          namespace: 'test-azure-ws-namespace',
+          workspace: mockAzureWorkspace,
+        })
+      );
+    });
+
+    await waitFor(() => {
+      expect(screen.getByRole('table')).toBeInTheDocument();
+    });
+
+    expect(screen.getByText('Submission statuses are all up to date.')).toBeInTheDocument();
+  });
+
+  it('should indicate incompletely updated polls', async () => {
+    const runSetData = _.merge(simpleRunSetData, { fully_updated: false });
+
+    const getRunSetsMethod = jest.fn(() => Promise.resolve(runSetData));
+    const mockLeoResponse = jest.fn(() => Promise.resolve(mockAzureApps));
+
+    await Ajax.mockImplementation(() => {
+      return {
+        Cbas: {
+          runSets: {
+            get: getRunSetsMethod,
+          },
+        },
+        Apps: {
+          listAppsV2: mockLeoResponse,
+        },
+      };
+    });
+
+    // Act
+    await act(async () => {
+      render(
+        h(BaseSubmissionHistory, {
+          name: 'test-azure-ws-name',
+          namespace: 'test-azure-ws-namespace',
+          workspace: mockAzureWorkspace,
+        })
+      );
+    });
+
+    await waitFor(() => {
+      expect(screen.getByRole('table')).toBeInTheDocument();
+    });
+
+    expect(screen.getByText('Some submission statuses are not up to date. Refreshing the page may update more statuses.')).toBeInTheDocument();
+  });
+
+  it('should abort successfully', async () => {
+    let content;
+    const user = userEvent.setup();
+    const runSetData = simpleRunSetData;
+    const getRunSetsMethod = jest.fn(() => Promise.resolve(runSetData));
+    const cancelSubmissionFunction = jest.fn(() => Promise.resolve(mockAbortResponse));
+    const mockLeoResponse = jest.fn(() => Promise.resolve(mockAzureApps));
+
+    await Ajax.mockImplementation(() => {
+      return {
+        Cbas: {
+          runSets: {
+            get: getRunSetsMethod,
+            cancel: jest.fn(cancelSubmissionFunction),
+          },
+        },
+        Apps: {
+          listAppsV2: mockLeoResponse,
+        },
+      };
+    });
+
+    // Act
+    await act(async () => {
+      content = render(
+        h(BaseSubmissionHistory, {
+          name: 'test-azure-ws-name',
+          namespace: 'test-azure-ws-namespace',
+          workspace: mockAzureWorkspace,
+        })
+      );
+    });
+
+    // Assert
+    await waitFor(() => {
+      expect(getRunSetsMethod).toBeCalled();
+    });
+
+    expect(content.getByRole('table')).toBeInTheDocument();
```

Comment:
These queries could probably use `screen` instead of `content`.

```suggestion
    expect(screen.getByRole('table')).toBeInTheDocument();
```

https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen